src/CppUTest/TestRegistry.cpp

```diff
@@ -50,6 +50,7 @@ void TestRegistry::runAllTests(TestResult& result)
     result.testsStarted();
     for (UtestShell *test = tests_; test != NULL; test = test->getNext()) {
```

Comment:
@basvodde , here we can see, in the TestRegistry, all test shell is taken to process one by one based on the UtestShell interface. so i need to add setRunIgnore in the UtestShell interface, or I have to segregate the UtestShell interface to get a few new classification